scripts/globals/quests.lua

```diff
@@ -33,6 +33,13 @@ dsp.quests.enums =
         SERVER_VAR = 3,
     },
 
+    quest_status =
+    {
+        QUEST_AVAILABLE = 0;
```

Comment:
comma instead of semicolon for new element in table

Reply:
Whoops, don't know how I didn't catch that. Surprised Lua didn't yell at me.

Reply:
depending on the lua interpret used it sometimes allows, still we prefer the comma for consistency :+1: